pkg/cli/cmd/bicep/publish/publish_test.go

```diff
@@ -48,13 +48,13 @@ func TestRunner_extractDestination(t *testing.T) {
 		},
 		{
 			name:    "no tag",
-			target:  "test.azurecr.io/test/repo",
+			target:  "ghcr.io/test-registry/test/repo",
```

Comment:
@ytimocin , @willdavsmith ,  Can we define all these path constants at one common place ghcr.io as root path , test-registry and similar in the code, so we need to change at one place. Currently since it is hard coded, we need to update almost all the places.

Reply:
can i do this as a fast follow? I'm running out of time here